.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -1,18 +1,32 @@
+---
 name: Report an issue
-description: Tell us about something that's not working the way we (probably) intend
+description:
+  Tell us about something that is not working the way we (probably) intend.
 body:
-  - type: input
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for contributing to Elixir.
+
+        Please, do not use this form for guidance, questions or support.
```

Comment:
I wonder if we can force a line break somehow? In any case, it is not a blocker!

Reply:
Maybe doubling up the new lines. I will look into it.

Reply:
@josevalim Let's merge this as it is, since i need to see the changes applied in my `main`  branch, it is easier for me that this one gets merged, and then I submit a second PR if possible.